test/acceptance/tests/hcp/hcp_test.go

```diff
@@ -615,7 +615,7 @@ func TestAuditLogging(t *testing.T) {
 		Namespace:    "default",
```

Comment:
nit: On line 600 (where I can't comment), you can also remove `"ecs_cluster_1_arn", "ecs_cluster_2_arn"` from the `ignoreVars`. These numbered cluster arns are no longer used by any of the Terraform modules, so we don't need to reference them (but having them in ignoreVars won't cause any issues either).